bin/genesis

```diff
@@ -139,7 +139,13 @@ setup() {
 		while IFS='' read -r line || [[ -n "$line" ]]; do
 			cmd=$(echo $line | perl -pe 's|^(.*?):.*$|$1|')
 			ver=$(echo $line | perl -pe 's|^.*?: *(.*)$|$1|')
-			if [[ -z $(command -v ${cmd}) ]]; then
+			if [[ ${cmd} == "genesis" ]] ; then
+				if [[ ${ver} != ${VERSION} ]] ; then
```

Comment:
We already support minimum version checking.  The change here should be to just use the value of `$VERSION` instead of running `genesis -v`.  We should still do the comparison down below.